policyengine_us/variables/gov/states/sc/tax/income/sc_fed_taxable_income.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class sc_fed_taxable_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "South Carolina federal taxable income exculde salt"
+    unit = USD
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        agi = tax_unit("adjusted_gross_income", period)
+        deductions_less_salt = tax_unit(
```

Comment:
This is only if they itemize right?

Reply:
Yes, this only shows the scenario where a household chose to use itemized deductions for their federal taxable income. Should I also show the scenario where a household chose to use standard deduction? something like 
`deductions = where("itemized_deductions_less_salt" > "standard_deduction", "itemized_deductions_less_salt", "standard_deduction") return max_(0, agi-deductions)`

Reply:
I think it's based on the federal itemization choice, which is `tax_unit_itemizes`

Reply:
you still need the standard deduction